projects/stingray/index.md

```diff
@@ -0,0 +1,17 @@
+
+![private](/images/privateresidence.png)
+
+After our main case moved its ways through the courts, we eventually [prevailed](https://lucyparsonslabs.com/foia/StingrayWin/) forcing the release of over 300 pages of documents on how CPD deploys Stingrays and what they tell courts. In particular, we have seen that CPD describes their use of Stingrays as “digital analysis”, which doesn’t fully portray the Constitutional or privacy implications of Stingray use. According to CPD’s description of “digital analysis” they never describe that Stingrays can capture the numbers of non-target phone numbers to courts, which is a major concern for civil liberties. As reported in [The Intercept](https://theintercept.com/2016/10/18/how-chicago-police-convinced-courts-to-let-them-track-cellphones-without-a-warrant/) “While these court records reflect how Stingrays and other cellphone trackers were authorized for use in serious criminal cases — such as murders, kidnappings, and aggravated battery — they also demonstrate how little judges knew about the devices prior to approval. The documents also demonstrate the ties surveillance technology has to the money and possessions seized during the war on drugs — which is in turn used to track people in impoverished areas of Chicago.” Just as troubling is that Pen Register / Trap and Trace orders (PR/TT) are allowing Stingrays to be used “including in private places, to determine the electronic identifying number of any cellular telephone being used by [REDACTED]” (see image above). Precisely for these reasons it is important to force police departments into greater transparency as we have done in Illinois. 
```

Comment:
moved its ways -> moved its way

important to force police departments into greater transparency  -> important to force greater transparency in police departments